app/views/start-id.html

```diff
@@ -36,15 +36,15 @@ <h1 class="govuk-heading-xl">
           Register for a national professional qualification (NPQ)
         </h1>
         <h3 class="govuk-heading-m">Registration has closed temporarily</h3>
-        <p><a href="/closed/eoi-email">Sign up for an email</a> to be notified when registration opens. </p>
+        <p><a href="/closed/eoi-email">Request an email to tell you when registration reopens</a>. </p>
         <p>When registration opens use this service to register for an NPQ or the early headship coaching offer. </p>
```

Comment:
Should this line move down into the bullets?

Reply:
Yeah, it makes sense to remove this line at the top: 'When registration opens use this service to register for an NPQ or the early headship coaching offer.' and have the same info as a bullet at the bottom of the page.